cloud/azure/actuators/machine/machineactuator_test.go

```diff
@@ -153,7 +218,62 @@ func TestBase64Encoding(t *testing.T) {
 }
 
 func TestGetStartupScript(t *testing.T) {
-	expectedStartupScript := "echo 'Hello world!'"
+	expectedStartupScript := `(
```

Comment:
Can place the startup script to a file and just read it from here? That way we don't have to do a copy and paste.

Reply:
I tried making this change but was running into some issues where creating the azure deployment would silently fail. Don't think I'll have time to work through it today.